packages/api/server.js

```diff
@@ -23,21 +23,25 @@ const GetTestHandler = require('./src/get-test.js');
 const GetExportHandler = require('./src/get-export.js');
 const client = require('./src/firestore.js');
 const auth = require('./src/auth.js');
+const { InvalidParameterError } = require('./lib/errors');
 
 const { v4 } = require('uuid');
 
 const cors = require('cors');
 
-global.headCollection = process.env.HEAD_COLLECTION || 'testing/main/repos';
+global.headCollection = 'production/main/repos';
 
 app.use(cors());
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: true })); // support encoded bodies
 
-app.delete('/api/repo/:orgname/:reponame/test/:testid', async (req, res) => {
+app.get('/api/repo/deleteurl/:orgname/:reponame/test', async (req, res) => {
   const orgName = req.params.orgname;
   const repoId = req.params.reponame;
-  const testName = req.params.testid;
+  if (!req.query.testname) {
+    throw new InvalidParameterError('Route requires query parameter of name');
```

Comment:
I like that you're starting to build some helpers for different types of errors.